### PR TITLE
20.9.0 rns

### PR DESCRIPTION
**Pull request by jochman**
## Status
In Progress

### CODE REVIEW

File: content-repo/extra-docs/releases/20.9.0.md [hunk 1]
```diff
@@ -0,0 +1,418 @@
+# Cortex XSOAR Content Release Notes for version 20.8.2 (98126)
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:1]: update version

File: content-repo/extra-docs/releases/20.9.0.md [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+##### Published on 1 September 2020
+### New: EmailCommunication Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:3]:
```suggestion
### New: Email Communication Pack v1.0.0
```

File: content-repo/extra-docs/releases/20.9.0.md [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+- **Microsoft ATA Suspicious Activity**
+#### Integrations
+##### Microsoft Advanced Threat Analytics  
+Use Microsoft Advanced Threat Analytics integration to manage suspicious activities, monitoring alerts and entities.
+
+
+---
+
+## New: NTT Cyber Threat Sensor Pack v1.0.0 (Partner Supported)
+#### Classifiers!-->
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:59]:
```suggestion
#### Classifiers
```

File: content-repo/extra-docs/releases/20.9.0.md [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+##### NTT Cyber Threat Sensor  
+Retrieve alerts and recommendations from NTT CTS
+#### Playbooks
+##### Handle TD events  
+Playbook to enrich TD events
+
+
+---
+
+### New: PiHole Pack v1.0.0
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:81]: this is community supported
@glicht do we want to add a comment about that?

File: content-repo/extra-docs/releases/20.9.0.md [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+##### CommonServerPython  
+Added the following code objects, which simplifies creating widgets.
+  - TextWidget
+  - TrendWidget
+  - NumberWidget
+  - BarColumnPieWidget
+  - LineWidget
+  - TableOrListWidget
+- Fixed an issue with mirroring mapper scheme.
+Fixed an issue in the **return_outputs** function where the content type was incorrect.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:118]:
```suggestion
- Fixed an issue in the **return_outputs** function where the content type was incorrect.
```

File: content-repo/extra-docs/releases/20.9.0.md [hunk 6]
```diff
@@ -0,0 +1,418 @@
...
+
+---
+
+### Base Pack v1.1.17
+#### Scripts
+##### WordTokenizerNLP  
+Updated the script docker image to the latest version.
+
+##### CommonServerPython  
+Added the following code objects, which simplifies creating widgets.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:110]:
```suggestion
- Added the following code objects, which simplifies creating widgets.
```

File: content-repo/extra-docs/releases/20.9.0.md [hunk 7]
```diff
@@ -0,0 +1,418 @@
...
+##### Code42  
+Fix bug where File Category would not map correctly when creating incidents from Code42 alerts.
+
+---
+
+### Cofense Triage Pack v1.1.4 (Partner Supported)
+#### Integrations
+##### Cofense Triage v2  
+- Fixed an issue in *cofense-get-attachment* command.
+- Internal code improvements.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:158]: lets remove that line

File: content-repo/extra-docs/releases/20.9.0.md [hunk 8]
```diff
@@ -0,0 +1,418 @@
...
+
+
+---
+
+### CyberArk Pack v1.0.2
+#### Integrations
+##### CyberArk PAS  
+- Added documentation for the integration.
+- Updated the the **cyberark-pas-credentials-verify** command to use the new API.
+Added the ***cyberark-pas-account-get-details*** command.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:202]:
```suggestion
- Added the ***cyberark-pas-account-get-details*** command.
```

File: content-repo/extra-docs/releases/20.9.0.md [hunk 9]
```diff
@@ -0,0 +1,418 @@
...
+##### IBM QRadar  
+Added a parameter that enables you to specify the number of addresses to enrich per API call.
+
+
+---
+
+### Infocyte Pack v1.0.2 (Partner Supported)
+#### Integrations
+##### Infocyte  
+Maintenance and stability enhancements.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:285]: lets remove that pack from the rn

File: content-repo/extra-docs/releases/20.9.0.md [hunk 10]
```diff
@@ -0,0 +1,418 @@
...
+
+##### New: ServiceNow - Incoming Mapper  
+Maps incoming ServiceNow incidents fields.
+
+#### Integrations
+##### ServiceNow v2  
+- Added the ticket mirroring feature.
+- Removed unnecessary import.
+
+#### Layouts
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:389]: need to add the layout here
@yaakovi any idea why it was generated empty ?

File: content-repo/extra-docs/releases/20.9.0.md [hunk 11]
```diff
@@ -0,0 +1,418 @@
...
+#### Integrations
+##### Symantec Endpoint Protection v2  
+Fixed an issue in which the *domain* parameter was not parsed correctly.
+
+---
+
+### Tanium Pack v1.0.2
+#### Integrations
+##### Tanium v2  
+- Added completion-percantage argument to the commands **tn-get-saved-question-result** and **tn-get-question-result**
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.9.0.md:411]:
```suggestion
Added completion-percantage argument to the commands **tn-get-saved-question-result** and **tn-get-question-result**
```

File: content-repo/extra-docs/releases/20.9.0.md (hunk 4 above)
Comment (glicht) [reply content-repo/extra-docs/releases/20.9.0.md:81]: Yes. We can add "Community Supported".

File: content-repo/extra-docs/releases/20.9.0.md (hunk 2 above)
Comment (glicht) [reply content-repo/extra-docs/releases/20.9.0.md:3]: We should also add a pack description for this one as it is a "use case" pack. We can copy the description from the pack:

> Do you have to send multiple emails to end users? This content pack helps you streamline the process and automate updates, notifications and more.

File: content-repo/extra-docs/releases/20.9.0.md [hunk 12]
```diff
@@ -0,0 +1,408 @@
...
+##### Genians  
+Use the Genians integration to block IP using assign tag and unassign tag.
+
+
+---
+
+### New: McAfee ESM Pack v1.0.0
+#### Integrations
+##### McAfee ESM v2  
+Run queries and receive alarms from Intel Security ESM. Support version 10 and above.
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:30]: McAfee is no long part of Intel.
```suggestion
Run queries and receive alarms from McAfee ESM. Support version 10 and above.
```

File: content-repo/extra-docs/releases/20.9.0.md [hunk 13]
```diff
@@ -0,0 +1,408 @@
...
+#### IncidentTypes
+- **Microsoft ATA Suspicious Activity**
+#### Integrations
+##### Microsoft Advanced Threat Analytics  
+Use Microsoft Advanced Threat Analytics integration to manage suspicious activities, monitoring alerts and entities.
+
+
+---
+
+## New: NTT Cyber Threat Sensor Pack v1.0.0 (Partner Supported)
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:58]: Problem with table of contents
```suggestion
### New: NTT Cyber Threat Sensor Pack v1.0.0 (Partner Supported)
```

File: content-repo/extra-docs/releases/20.9.0.md [hunk 14]
```diff
@@ -0,0 +1,408 @@
...
+#### Integrations
+##### Query.AI  
+Query.AI is a decentralized data access and analysis technology that simplifies security investigations across disparate platforms without data duplication.
+
+
+---
+
+### Access Investigation Pack v1.2.2
+#### Layouts
+- **layout-edit-Access.json**
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:99]: @jochman @Itay4 What changed here?

File: content-repo/extra-docs/releases/20.9.0.md [hunk 15]
```diff
@@ -0,0 +1,408 @@
...
+### Expanse Pack v1.1.1 (Partner Supported)
+#### Integrations
+##### Expanse  
+Updated the version number in the user-agent header
+
+---
+
+### GDPR Pack v1.0.5
+#### Layouts
+- **layout-edit-GDPR_Data_Breach.json**
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:253]: What changed here?

File: content-repo/extra-docs/releases/20.9.0.md [hunk 16]
```diff
@@ -0,0 +1,408 @@
...
+#### Layouts
+- **layout-details-IronDefense_IronDome_Notification.json**
+- **layout-details-IronDefense_Event_Notification.json**
+- **layout-details-IronDefense_Alert_Notification.json**
+
+---
+
+### Malware Pack v1.2.3
+#### Layouts
+- **layout-edit-Malware.json**
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:305]: What changed?

File: content-repo/extra-docs/releases/20.9.0.md [hunk 17]
```diff
@@ -0,0 +1,408 @@
...
+### PassiveTotal Pack v1.0.1
+#### Integrations
+##### PassiveTotal  
+Added a limit to the number of related domains showed in the war room for enrichment commands.
+
+---
+
+### Phishing Pack v1.10.2
+#### Layouts
+- **layout-quickView-Phishing.json**
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:348]: What changed?

File: content-repo/extra-docs/releases/20.9.0.md [hunk 18]
```diff
@@ -0,0 +1,408 @@
...
+Fixed an issue where `WhereFieldEquals` would return a string instead of a list.
+
+##### FeedRelatedIndicatorsWidget  
+Fixed an issue where the indicator link value was incorrect.
+
+---
+
+### Common Types Pack v1.8.8
+#### Layouts
+- **File Indicator**
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:180]: what changed?

File: content-repo/extra-docs/releases/20.9.0.md [hunk 19]
```diff
@@ -0,0 +1,408 @@
...
+
+### DUO Admin Pack v2.0.1
+#### Integrations
+##### DUO Admin  
+Updated the integration docker image to the latest version.
+
+---
+
+
+### Email Communication Pack v1.2.0
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.9.0.md:214]: We have this at the top as a `New` pack. Please unite.

**Review by glicht — CHANGES_REQUESTED** [review]

File: content-repo/extra-docs/releases/20.9.0.md (hunk 17 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.9.0.md:348]: Need to investigate why it happens on layouts only.

**Review by Itay4 — APPROVED** [review]

**Pull request merged**